src/main/java/com/salesforce/dynamodbv2/mt/mappers/sharedtable/impl/MtAmazonDynamoDbBySharedTable.java

```diff
@@ -497,6 +545,22 @@ public UpdateItemResult updateItem(UpdateItemRequest updateItemRequest) {
         return getAmazonDynamoDb().updateItem(updateItemRequest);
     }
 
+    @Override
+    public RestoreTableFromBackupResult restoreTableFromBackup(
+        RestoreTableFromBackupRequest restoreTableFromBackupRequest) {
+        throw new UnsupportedOperationException();
```

Comment:
these are still throwing unsupported operation exceptions, so doing nothing at the moment, but these are the hooks I imagined clients of mt-dynamo will use to consume this backup/restore support. 

I figure, all we need is a translation of these dynamo restore/backup requests, to the simpler backup/restore request objects defined in `MtBackupManager.kt` and vice versa for the result objects. Something close to my next todos, but I figure, comes secondary to any review feedback of this and ancillary PRs